src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelGene.tsx

```diff
@@ -171,11 +164,14 @@ const TrackPanelGene = (props: TrackPanelGeneProps) => {
           </span>
         </div>
       </GroupTrackPanelItemLayout>
-      {sortedTranscripts.map((transcript) => (
+      {visibleSortedTranscripts.map((transcript) => (
         <TrackPanelTranscript
           transcript={transcript}
           genomeId={genomeId}
-          focusObjectId={focusObjectId}
+          isVisible={
+            visibleTranscriptIds?.includes(transcript.stable_id) ?? false
+          }
+          onVisibilityChange={onTranscriptVisibilityChange}
```

Comment:
Dont we need to pass the transcriptId and isVisible to this function?

Reply:
`TrackPanelTranscript` does so when it calls `props.onVisibilityChange`

Reply:
aah yes, this is being passed to child component! doh!